src/Kernel/client/kernel.ts

```diff
@@ -21,9 +21,56 @@ class Kernel {
             this.requestEcho();
             this.requestClientInfo();
             this.setupMeasurementHistogramDataListener();
+            this.setupMeasurementHistogramDebugger();
         });
     }
 
+    setupMeasurementHistogramDebugger() {
+        IPython.notebook.kernel.register_iopub_handler(
+            "iqsharp_debug_opstart",
+            message => {
+                console.log("iqsharp_debug_opstart message received", message);
+
+                //check to see if there's already a button
+
+                let state: DisplayableState = message.content.state;
+                let state_div = state.div_id;
+                if (state_div != null) {
+                    let div = document.getElementById(state_div);
+                    if (div != null) {
+                        //handle displaying dumpMachine stuff here
+                        if (document.getElementById("ye") == null) {
```

Comment:
What does `ye` mean in this case? Or is it a placeholder?

Reply:
It's a placeholder, sorry!

Reply:
No worries haha! Just wanted to check in case you forget to update it :)